src/views/anon/intro/WhatWeOffer.tsx

```diff
@@ -0,0 +1,192 @@
+                        Beyond a co-working space
+                        </Typography>
+                        <Typography sx={{display: {xs: 'none', xl: 'inline'}}} variant='h3' gutterBottom>
+                        Beyond a co-working space
+                        </Typography>
+
+                        <Typography variant='subtitle1'>
+                        Venture Build is more than a traditional co-working space, it’s a collaborative ecosystem that includes tailored various features tailored to supercharge you startup journey.
+                        </Typography>
+                    </Grid>
+                    <Grid item lg={6} sx={{height: '100%', width: '100%'}}>
+                        <Box sx={{marginBottom: '20px', paddingRight: '15px'}}>
+                            <Typography variant='h5' fontWeight='bold'>
+                            A platform for smooth growth tracking
+                            </Typography>
+                            <Typography variant='subtitle1'>
+                            With intuitive tools and growth analytics, we simplify the path to monitor your venture growth, allowing for a new level of precision in tracking your growth milestones with us.
+                            </Typography>
+                        </Box>
+                        <Box>
+                            <Typography variant='h5' fontWeight='bold'>
+                            A platform for tailored support
+                            </Typography>
+                            <Typography variant='subtitle1'>
+                            Through regular needs assessments and our algorithm-powered recommendation system, we tailor our support to your unique requirements. From mentorship to curated resources, count on us to guide you through your journey with precision and care.
+                            </Typography>
+                        </Box>
+                    </Grid>
+                </Grid>
+
+                <Container maxWidth='xl' sx={{paddingTop: '40px', paddingBottom: '0px', display: 'flex', justifyContent: 'center'}}>
+                    <img src={video} alt={'video'} style={{padding: '10px'}} />
+                </Container>
+            </Box>
+
+            <Box>
+                <Box sx={{overflowX: 'hidden', marginTop: '20px', display: 'flex', alignItems: 'center', flexDirection: 'column', textAlign: 'center'}}>
+                    <Typography variant='overline' fontWeight='bold' color='#B54708'>
+                        Features
+                    </Typography>
+                    <Typography sx={{display: {xs: 'inline', xl: 'none'}}} variant='h3' gutterBottom>
+                        Supercharge your startup experience
+                    </Typography>
+                    <Typography sx={{display: {xs: 'none', xl: 'inline'}}} variant='h3' gutterBottom>
+                        Supercharge your startup experience
+                    </Typography>
```

Comment:
These have exact same content and styling, so I don't think we need to vary over display sizes